notebooks/2.single-cell-compound-prio/4.selecting_hits.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+
+# ## Identifying hits
+#
+# In this section, we analyze the generated scores from the previous step. These scores quantify the magnitudes of phenotypic differences between the "on" and "off" morphological signatures. The "on" morphological signatures represent the morphology features that are significantly affected when compared to a reference state. In contrast, the "off" morphological signatures are features that remain unaffected by both states, making them a reliable source for tracking off-target effects.
```

Comment:
Consider including the context for what is a "reference state" for the CFReT data.